pic32mz.ld

```diff
@@ -19,6 +19,7 @@ SECTIONS
 
   .text : AT(ORIGIN(flash))
   {
+    _text = ABSOLUTE(.);
```

Comment:
Please note that C compiler converts `name` symbol to `_name` linker symbol. Thus by a mere mistake you may end up using `text` in C but refer to the beginning of text section instead of just a variable.

If you have a Debian based distro look at linker script in `/usr/lib/ldscripts/elf_x86_64.x` file. Most of the time they use double underscore for special linker symbols.